### PR TITLE
Gh 1908 parquet store readme is out of date

### PR DESCRIPTION
**Pull request by gaffer01**
Merge after https://github.com/gchq/Gaffer/pull/2059

### CODE REVIEW

**Comment by gaffer01** [comment]
This was forked from PR #2059. Once that has been merged, this PR will show that only the README changed.

File: store-implementation/parquet-store/README.md [hunk 1]
```diff
@@ -56,19 +73,22 @@ This store can be used as a source of data for frameworks such as GraphX, e.g. t
...
 
 - `spark.master`: The string that sets what mode to run Spark in. By default, if Spark is installed on the machine it will use Spark's defaults, otherwise it will run in local mode using all available threads;
-- `parquet.data.dir`: The file path to save the graph files under, by default this will be a relative path \<current path\>/parquet_data;
-- `parquet.temp_data.dir`: The file path to save the temporary graph files under, by default this will be a relative path \<current path\>/.gaffer/temp_parquet_data. Warning: this directory will automatically be deleted at the start and end of any `AddElements` operation;
-- `parquet.threadsAvailable`: The number of threads to make available to the operations to increase the parallelism, by default this is set to 3 which will provide maximum parallelism when adding a single Gaffer group;
+- `parquet.data.dir`: The directory to use to save the graph;
```
Comment (d47853) [inline store-implementation/parquet-store/README.md]:
```suggestion
- `parquet.data.dir`: The directory used to save the graph;
```

File: store-implementation/parquet-store/README.md [hunk 2]
```diff
@@ -12,39 +12,56 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
...
+1. [Notice](#notice)
+2. [Introduction](#introduction)
+3. [Design overview](#design-overview)
+4. [Use cases](#use-cases)
+5. [Parquet version](#parquet-version)
+6. [Properties file](#properties-file)
 4. [Schema](#schema)
 5. [Inserting data](#inserting-data)
 6. [Queries](#queries)
+7. [Custom serialisers](##Writing a custom serialiser)
```
Comment (d47853) [inline store-implementation/parquet-store/README.md]:
```suggestion
7. [Custom serialisers](#custom-serialisers)
```

File: store-implementation/parquet-store/README.md [hunk 3]
```diff
@@ -12,39 +12,56 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
...
 7. [Optimisations](README.md#optimisations)
```
Comment (d47853) [inline store-implementation/parquet-store/README.md]:
```suggestion
7. [Optimisations](#optimisations)
```

File: store-implementation/parquet-store/README.md [hunk 4]
```diff
@@ -102,7 +118,7 @@ Graph graph = new Graph.Builder()
       .build();
 ```
 
-To get the best performance you should allow Gaffer to detect the best serialiser or provide a serialiser class that implements `ParquetSerialisation`.
+To get the best performance you should allow Gaffer to detect the best serialiser or provide a serialiser class that implements `ParquetSerialisation`. See the section on serialisers in this README for further information about serialisers optimised to work with the Parquet store.
```
Comment (d47853) [inline store-implementation/parquet-store/README.md]:
```suggestion
To get the best performance you should allow Gaffer to detect the best serialiser or provide a serialiser class that implements `ParquetSerialisation`. See the section on [serialisers](custom-serialisers) for further information about serialisers optimised to work with the Parquet store.
```

**Pull request merged**